.gitignore

```diff
@@ -33,4 +33,5 @@ Thumbs.db
 .AppleDouble
 .LSOverride
 .Spotlight-V100
-.Trashes
\ No newline at end of file
+.Trashes
+package-lock.json
```

Comment:
I think it would be better to set `package-lock=false` on .npmrc so the lock file would not be generated at all